src/coreclr/tests/src/Common/Directory.Build.targets

```diff
@@ -29,16 +29,49 @@
       <RunTimeDependencyCopyLocalFile Include="@(AllResolvedRuntimeDependencies)"  Exclude="@(RunTimeDependencyExcludeFiles)"/>
       <RunTimeDependencyCopyLocal Include="@(RunTimeDependencyCopyLocalFile -> '%(File)')"  />
       <RunTimeDependencyCopyLocal Include="$(TargetingPackPath)/*" />
-      
-      <RunTimeArtifactsExcludeFiles Include="protononjit.dll" />
-      <RunTimeArtifactsExcludeFiles Include="linuxonjit.dll" />
-      
+    </ItemGroup>
+
+    <ItemGroup>
+      <RunTimeArtifactsExcludeFiles Include="PDB/createdump.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/dbgshim.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/linuxonjit.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/mcs.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/mscordaccore.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/mscordbi.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/mscorrc.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/protononjit.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/superpmi.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/superpmi-shim-collector.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/superpmi-shim-counter.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/superpmi-shim-simple.pdb" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(MinimalCoreRoot)' == 'true'">
+      <RunTimeArtifactsExcludeFiles Include="PDB/ilasm.pdb" />
+      <RunTimeArtifactsExcludeFiles Include="PDB/ildasm.pdb" />
+    </ItemGroup>
+
+    <ItemGroup>
       <RunTimeArtifactsIncludeFolders Include="/" />
+
+      <!-- Used by the Crossgen comparison job -->
       <RunTimeArtifactsIncludeFolders Include="IL/" />
+
+      <!-- Used for Crossgen2 R2R tests -->
       <RunTimeArtifactsIncludeFolders Include="crossgen2/" />
+
+      <!-- Used for capturing symbolic stack traces using Watson -->
       <RunTimeArtifactsIncludeFolders Include="PDB/" />
+
+      <!-- Used by the coreroot_determinism test -->
       <RunTimeArtifactsIncludeFolders Include="R2RTest/" />
 
+      <!-- Used for ARM / ARM64 cross-compilation -->
+      <RunTimeArtifactsIncludeFolders Include="x86/" />
```

Comment:
I have a feeling we should use the bin folder for these cross arch crossgen usages and keep these removed. @echesakovMSFT opinions?

Reply:
I don't have strong opinion here - I am fine with not including these in Core_Root as long as crossgen_comparison tests continue working.

Reply:
crossgen comparison already uses the bin directory to my knowledge. There was something that broke with the removal. @BruceForstall what was that?

Reply:
I found it: https://github.com/dotnet/runtime/pull/35243#issuecomment-620407386

Reply:
https://github.com/dotnet/runtime/pull/35577 addresses this issue and I believe the only usage.